node/src/chain_spec.rs

```diff
@@ -192,6 +193,13 @@ pub fn testnet_genesis(
             capsule_mint_fee: 1000000000000000000000,
             ..Default::default()
         },
+        altvr: AltvrConfig {
+            users: vec![(
```

Comment:
Is there are reason why we don't use `users: Default::default()` ?

Reply:
done